internal/node/node_repositories.bzl

```diff
@@ -239,21 +239,10 @@ alias(name = "npm", actual = "{npm}")
 alias(name = "yarn", actual = "{yarn}")
 filegroup(
   name = "node_runfiles",
```

Comment:
in your change, you introduced a new `minimal_node_runfiles` but then I believe the existing one became unreferenced.

It's possible that users downstream referenced `@npm//:node_runfiles` and rely on the old content, we should figure that out by vetting this PR on downstream repos.